src/qt/intro.cpp

```diff
@@ -346,3 +329,25 @@ QString Intro::getPathToCheck()
     mutex.unlock();
     return retval;
 }
+
+void Intro::UpdatePruneLabels(uint64_t prune_target_gb)
+{
+    m_required_space_gb = m_blockchain_size;
+    QString storageRequiresMsg = tr("At least %1 GB of data will be stored in this directory, and it will grow over time.");
+    if (ui->prune->isChecked()) {
```

Comment:
In commit "refactor: Add Intro::UpdatePruneLabels()" (27f716ebbadfefee8eac7e43374f080ff3dbc68a)

Why did this line change from `if (prune_target_gb)` if this commit is supposed to be move-only?

Reply:
> Why did this line change from `if (prune_target_gb)` if this commit is supposed to be move-only?

You are right. It is a remnant of the previous branch. Will be fixed.

Reply:
Fixed.